src/sentry/static/sentry/app/components/group/tagDistributionMeter.jsx

```diff
@@ -113,7 +113,7 @@ const TagDistributionMeter = React.createClass({
                 .tag}/`}
               title={
                 '<div class="truncate">' +
-                escape(deviceNameMapper(value.name)) +
+                escape(deviceNameMapper(value.name) || '') +
```

Comment:
i also could do something like `(none)` to make it more clear that no name was provided ?

Reply:
Can you add a screenshot?

Reply:
should this go inside devicenamemapper?

Reply:
before fixing:
![screenshot 2017-11-01 11 04 11](https://user-images.githubusercontent.com/5026776/32289777-f1a92f2a-bef4-11e7-897e-3509eea1cf97.png)

after fixing:
![screenshot 2017-11-01 11 06 00](https://user-images.githubusercontent.com/5026776/32289792-017bd362-bef5-11e7-849b-b4c638f9e844.png)

what it looks like with all data:
![screenshot 2017-11-01 11 06 19](https://user-images.githubusercontent.com/5026776/32289802-0a9cab2e-bef5-11e7-955e-b8f4c447278e.png)

`deviceNameMapper` seems pretty explicit in it's returning null when the value is falsey so I'm hesitant to edit that in case it will cause problems elsewhere: https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/utils/deviceNameMapper.jsx#L5

Reply:
maybe `n/a`? if there are n tags/devices would it list all n?

Reply:
The thing that is broken is what lists the name with the highest percent (when the name is an empty string) -- so I could replace it with `(none)` or `(empty)` or something. Or just leave as is and it's a little weird, but better than breaking the entire component.